src/test/java/se/sundsvall/casedata/service/scheduler/supensions/SuspensionWorkerTest.java

```diff
@@ -0,0 +1,87 @@
+package se.sundsvall.casedata.service.scheduler.supensions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import se.sundsvall.casedata.api.model.PatchErrand;
+import se.sundsvall.casedata.integration.db.ErrandRepository;
+import se.sundsvall.casedata.integration.db.model.ErrandEntity;
+import se.sundsvall.casedata.service.ErrandService;
+
+
+@ExtendWith(MockitoExtension.class)
+class SuspensionWorkerTest {
+
+	@Mock
+	private ErrandRepository errandsRepository;
+
+	@Mock
+	private ErrandService errandService;
+
+	@InjectMocks
+	private SuspensionWorker suspensionWorker;
+
+	@Captor
+	private ArgumentCaptor<PatchErrand> errandCaptor;
+
+	@Test
+	void cleanUpSuspensions() {
```

Comment:
Shouldn't this test also assert that the errandEntity no longer have "suspendedFrom" and "suspenededTo"? Or am I missing something?

Reply:
But errandEntity does have `suspendedFrom` and `suspenededTo` still. The `patchedEntity` and the `entityId` is passsed to the `errandService`for updating. Thus the updating and removal of `suspendedFrom` and `suspenededTo` happens outside the bounds of this test.

Reply:
And before you ask the The `patchedEntity` and the `entityId` is passed to the errandService for updating because we want to jack into the notification that should be sent when a `errandEntity` is updated.